docs/ingester-handover.md

```diff
@@ -0,0 +1,44 @@
+# Ingester Hand-over
+
+The [ingester](architecture.md#ingester) holds several hours of sample
+data in memory. When we want to shut down an ingester, either for
+software version update or to drain a node for maintenance, this data
+must not be discarded.
+
+Each ingester goes through different states in its lifecycle. When
+working normally, the state is `ACTIVE`.
```

Comment:
May want to mention these states can be seen on the /ring page on the querier and distributor.